### DISCUSSION TITLE
Should the Amazon camera image update?

### DISCUSSION THREAD

**Discussion by riddledaxis** [moralmunky/Home-Assistant-Mail-And-Packages#438] [Q&A] [answered]
I received an Amazon delivery yesterday, and the sensors picked up that the item was out for delivery and then that it had been delivered. But the camera image never changed all day. It was being updated, as the checked date and time changed as you would expect throughout the day but the image didn't.

Is this still a work in progress, or should it have changed?

I am running M+P beta version 0.3.2-b11, HassOS 5.13, core-2021-4-0 and supervisor-2021-03-9.

![image](https://user-images.githubusercontent.com/47009467/114277311-96881b80-9a22-11eb-9f39-d061f6cca608.png)
![image](https://user-images.githubusercontent.com/47009467/114277326-9f78ed00-9a22-11eb-9072-89c573f8b8d7.png)

**Comment by firstof9** [comment, marked as answer]
It only changes if they send you an email with the picture in it.
If they hand you the package, they don't snap an image.
_Usually_ the driver snaps an image if they're just setting and leaving.

**Reply by riddledaxis** [reply]
oh, ok

Thank you
